theme-data/dark/dark-common.json

```diff
@@ -97,6 +97,9 @@
         },
         "tertiary":{
           "normal":"@color-gray-90"
+        },
+        "quartinary":{
```

Comment:
In the spec it's quatrinary, in your change quartinary. The actual word is quaternary. Can you follow up with the design team to correct the spelling please?

Reply:
yep :) just asked the change in the Figma